experimental/trusted_shuffler/trusted_shuffler/src/lib.rs

```diff
@@ -48,12 +49,16 @@ impl TrustedShufflerRequest {
 
 #[derive(Debug, PartialEq)]
 pub struct TrustedShufflerResponse {
-    pub body: Vec<u8>,
+    pub data: hyper::body::Bytes,
+    pub trailers: hyper::HeaderMap,
 }
 
 impl TrustedShufflerResponse {
     fn empty() -> TrustedShufflerResponse {
```

Comment:
Isn't this basically Default? If so, can it be derived automatically?

Reply:
Ehm, good point, but I'd like to keep it that way. When we have fixed size responses, the empty response should have a fixed size, too.